Haskell Day 2019向けのCode Of Conductを作成

igrep opened the pull request:
https://github.com/haskell-jp/community/issues/23 への対応。
概ねScalaMatsuriのもの https://2019.scalamatsuri.org/ja/code-of-conduct/ からのコピペです。
突貫工事になってしまいましたが、何人かに見ていただいた上で、マージし、イベントページにリンクを張り次第クローズします。

igrep committed:
Copy-and-pandoc from https://2019.scalamatsuri.org/ja/code-of-conduct/

igrep committed:
Haskell Day 2019の事情に合わせて改変

- 大半の箇所において「ScalaMatsuri」を「Haskell Day 2019」に変更
- 報告フォームや動画など、Haskell Day 2019で用意していないものは削除
    - 報告フォームの代わりにメールアドレスを用意
- ScalaMatsuriのCOCから持ってきたことを明記
- その他、改行位置の調整

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,54 @@
+--------
+
+[Haskell Day 2019](https://techplay.jp/event/727059) は、様々な地域やコミュニティから集う技術者に対して開かれたカンファレンスを目指しています。
+そのためには、性別や人種など多様な背景を持つ、普段の生活では会わない人々同士でも、互いに敬意を払って楽しい時間を過ごせることが重要だと考えています。
```
本筋ではないこまいところなんですが、空行なしの改行だと空白の入った1行になるので、空白が気になります。

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,54 @@
+
+ハラスメント行為の一例には以下のようなものがあります:
+
+-   他の参加者に対するナンパ行為(容姿に関する発言、恋愛・性的興味を目的とした発言)や不適切な身体的接触を行うこと
```
全角括弧と半角括弧の表記揺れ。個人的には和文だと全角括弧の方が好き。

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,54 @@
+
+自分や他の人がハラスメントを受けた場合には(TODO: スタッフの確認方法)に直接ご連絡いただくか、[haskell-jp-admins@googlegroups.com](mailto:haskell-jp-admins@googlegroups.com)までご連絡ください。
+
+### ライセンスと帰属に関して
```
「報告窓口」の子要素じゃないように思えるので、第2レベルがいいと思います。

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,54 @@
+報告窓口
+--------
+
+自分や他の人がハラスメントを受けた場合には(TODO: スタッフの確認方法)に直接ご連絡いただくか、[haskell-jp-admins@googlegroups.com](mailto:haskell-jp-admins@googlegroups.com)までご連絡ください。
```
メールアドレスの後に空白がない（他の欧文和文の間にはある）。

kakkun61 reviewed (changes requested):
全体的にこまかいところですが

maoe commented:
LGTM. Thank you!

igrep committed:
Fix formatting issues

- Avoid spaces inserted by newline in Markdown.
- Use double-width parentheses consistently.
- Put spaces before and after the mail address.

igrep committed:
Adjust the level of the section

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,55 @@
+報告窓口
+--------
+
+自分や他の人がハラスメントを受けた場合には(TODO: スタッフの確認方法)に直接ご連絡いただくか、 [haskell-jp-admins@googlegroups.com](mailto:haskell-jp-admins@googlegroups.com) までご連絡ください。
```
句読点自体にアキがあるので句読点と欧文の間はスペースなしにしたいです。

統一されていればいいのですが、「採用されており、Haskell Day 2019 もその精神に」やその他はスペースがないので……

```suggestion
自分や他の人がハラスメントを受けた場合には(TODO: スタッフの確認方法)に直接ご連絡いただくか、[haskell-jp-admins@googlegroups.com](mailto:haskell-jp-admins@googlegroups.com) までご連絡ください。
```

kakkun61 commented on haskell-day-2019/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,55 @@
+
+本規範のテキスト部分は [Creative Commons Zero license](https://creativecommons.org/publicdomain/zero/1.0/) のもとに公開します。パブリック・ドメイン同様に、派生して作られた規範に一切のクレジット表示やオープンライセンスなどを要求しません。
+
+本規範は [ScalaMatsuri](http://scalamatsuri.org/) の規範を元に、 Haskell Day 2019 固有の事情に合わせて改変しました。ScalaMatsuri の規範は Geek Feminism wiki の規範例から派生しており、PNW Scala、NE Scala、および Scala Days の影響を受けています。
```
```suggestion
本規範は [ScalaMatsuri](http://scalamatsuri.org/) の規範を元に、Haskell Day 2019 固有の事情に合わせて改変しました。ScalaMatsuri の規範は Geek Feminism wiki の規範例から派生しており、PNW Scala、NE Scala、および Scala Days の影響を受けています。
```

kakkun61 reviewed (changes requested):
またまたこまいですが…… :bow:

suggestion にしてるのでポチポチとお願いします :bow:

igrep committed:
Update haskell-day-2019/CODE_OF_CONDUCT.md

よく気づくなー。すごい

Co-Authored-By: Kazuki Okamoto <kazukiokamoto@iij.ad.jp>

igrep committed:
Update haskell-day-2019/CODE_OF_CONDUCT.md

Co-Authored-By: Kazuki Okamoto <kazukiokamoto@iij.ad.jp>

igrep committed:
スタッフの確認方法を追記

igrep commented:
皆さんレビューありがとうございました！
![IMG_20180408_093101](https://user-images.githubusercontent.com/227057/67830020-8c1ede00-fb1c-11e9-8753-661ea0bfdb9b.jpg)
